CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 ## v3.0.1 (2018-06-07)
 - Fixed version const
-
+- Add functionality to not invalidate cache when upsert (#341)
```

Comment:
3.0.1 has already been released, please make a new section for 3.0.2 and put your change there.

```
# Changelog

## v3.0.2 (unreleased)
 - Add functionality to not invalidate cache when upsert (#341)

## v3.0.1 (2018-06-07)
 - Fixed version const
...
```